clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonLoadBalancerProvider.groovy

```diff
@@ -63,15 +63,19 @@ class AmazonLoadBalancerProvider implements LoadBalancerProvider<AmazonLoadBalan
     CacheData application = cacheView.get(APPLICATIONS.ns, Keys.getApplicationKey(applicationName))
 
     def applicationServerGroups = application ? resolveRelationshipData(application, SERVER_GROUPS.ns) : []
+    def allLoadBalancerKeys = cacheView.getIdentifiers(LOAD_BALANCERS.ns)
```

Comment:
Only downside to that is it'll miss questionably named load balancers, i.e. ASG `fooproxy-v001` uses ELB `foo-frontend`